javascript/ql/src/semmle/javascript/security/dataflow/ReflectedXss.qll

```diff
@@ -53,6 +53,13 @@ module ReflectedXss {
     }
   }
 
+  /** A source of file name input, considered as a flow source for reflected XSS. */
```

Comment:
The phrase "source of file name input" sounds quite awkward. Maybe just "a file name"?